chord_metadata_service/discovery/model_lookups.py

```diff
@@ -19,3 +19,18 @@
     "biosample": DATA_TYPE_PHENOPACKET,
     "experiment": DATA_TYPE_EXPERIMENT,
 }
+
+PUBLIC_MODEL_NAMES_TO_SCOPE_FILTERS = {
+    "individual": {
+        "project": "phenopackets__dataset__project__identifier",
```

Comment:
looking at these deeply nested joins with the django ORM, it seems like it could introduce n+1 query issues... have you looked at if this is happening?

Reply:
Added some "prefetch_related" to avoid n+1